en/services/gimbal_v2.md

```diff
@@ -112,15 +112,27 @@ Every *Gimbal Manager* must to publish its associated *Gimbal Device* (there is
 
 A particular MAVLink component can implement multiple gimbal managers (e.g. an autopilot can implement two gimbal managers in order to control two gimbal devices).
 
+A gimbal manager can "manage" MAVLink gimbal devices as well as non-MAVLink gimbals such as a gimbal connected with proprietary or custom interface such as PWM or SBUS. A non-MAVLink gimbal is signalled and addressed using 1 to 6 as the `gimbal_device_id` instead of the MAVLink component ID.
 
-#### Addressing of Gimbal Devices {#gimbal_device_addressing}
+#### Addressing of Gimbal MAVLink Devices {#gimbal_device_addressing}
 
 *Gimbal Manager* commands and messages have a param field to indicate the component ID of the *Gimbal Device* that they intend to control.
 
 A system that wants to control a *particular* gimbal device will send messages to the component that has the manager(s), specifying the particular device to be controlled.
 
 If all gimbal devices should be controlled (on the component that has the gimbal managers), this param/field can be set to 0 (signalling "all").
 
+#### Addressing of non-MAVLink gimbal devices  {#non_mavlink_gimbal_device_addressing}
+
+Non-MAVLink gimbal devices are gimbal that don't expose the MAVLink API but instead are connected to the gimbal manager using some other protocol. For instance, this could be a PWM gimbal connected to an autopilot.
+
+For these cases, there needs to be a way to address such a gimbal specifically, and a way to send out the `GIMBAL_DEVICE_` messages, so that they can re mapped back to the respecive gimbal manager.
+
+The solution chosen for this case is to use the numbers 1 to 6 as magic numbers for the `gimbal_device_id`. This means that the numbers 1 to 6 can't be used as MAVLink component IDs for any components involved as gimbal managers or gimbal devices.
+
+- A **gimbal manager** advertises that it implements the gimbal device "itself" by setting `gimbal_device_id` to 1 to 6. It will also send out the requested gimbal device messages from the same component ID. It will set the field in `gimbal_device_id` of [GIMBAL_DEVICE_ATTITUDE_STATUS](#GIMBAL_DEVICE_ATTITUDE_STATUS).
+
+- A **ground station** addresses a gimbal device by sending commands to the gimbal manager and specifiying the `gimbal_device_id` 1 to 6.
```

Comment:
```suggestion
Non-MAVLink gimbal devices are gimbals that don't expose the MAVLink API, but instead are connected to the gimbal manager using some other protocol.
For instance, this could be a PWM gimbal connected to an autopilot.

For these cases, there needs to be a way to address such a gimbal specifically, and a way to send out the `GIMBAL_DEVICE_` messages, so that they can re mapped back to the respective gimbal manager.

The solution chosen for this case is to use the numbers 1 to 6 as magic numbers for the `gimbal_device_id`. This means that the numbers 1 to 6 can't be used as MAVLink component IDs for any components involved as gimbal managers or gimbal devices.

- A **gimbal manager** advertises that it implements the gimbal device "itself" by setting `gimbal_device_id` to 1 to 6. It will also send out the requested gimbal device messages from the same component ID. It will set the field in `gimbal_device_id` of [GIMBAL_DEVICE_ATTITUDE_STATUS](#GIMBAL_DEVICE_ATTITUDE_STATUS).

- A **ground station** addresses a gimbal device by sending commands to the gimbal manager and specifying the `gimbal_device_id` 1 to 6.
```